### DISCUSSION TITLE
Caresens air cgm integration

### DISCUSSION THREAD

**Discussion by Ylsani** [NightscoutFoundation/xDrip#3099] [Ideas]
Hi guys,

This is only way I could figure how to reach anyone. I would like to use xdrip, friend is using it and it seems super user friendly. However, I am using new (like, came on market few weeks ago new) cgm called caresens air. This cgm is right now only available in South Korea so I know there likely isn't much demand, but it should be on global markets soon-ish, so I am just throwing it out there, it would be cool to get it connected to xdrip! :) 

Please lemme know if there is any way to help with integrating, or lemme know if it will absolutely have to wait until it's available everywhere :)

**Comment by Navid200** [comment]
Please provide a link to the product.

Either someone who has access to the device and knows Java/Kotlin will open a PR to integrate it, or samples need to be sent to developers so that they can if possible.

**Reply by Ylsani** [reply]
Hi! Here is the cgm link: https://caresensair.com/en_US/

Store link: https://caresensmall.kr/mall/m_mall_detail.php?ps_ctid=&ps_goid=239&ps_page=

I hope there is a korean t1 who has this device and knows java. I don't :(

**Reply by Navid200** [reply]
Do you know if insurance companies in South Korea cover it?
Do they cover Dexcom G7?

**Reply by Ylsani** [reply]
Insurance in Korea does cover it at 70%(all cgms are covered at 70% of full price). G7 is not available here yet and not even a peep about when it will be. G6 is covered. Libre1 is latest libre available, libre2 should come late this year, libre3 is not expected anytime soon. Guardian4 is available. 

Libre and dex6 are roughly 100.000won full price per sensor. Caresens is 85000. Guardian is same as g6 I think but not sure. G6 transmitter is free of charge since few months ago.

Edit: I would happily get extras if I could. Unfortunately, insurance covers exactly number of sensors you need and not a single one more. So for 90mos, you get 6 sensors of caresens, libre is presribed at 7sensors for 98days and dexcom 9 sensors for 90days. Companies replace faulty ones for free, but you can't get insurance coverage for extras (you can buy them at full price though, cgms don't need prescription)

**Comment by jcisml-G59** [comment]
I just learned about the CareSens Air and wonder how it has been working for you, accuracy, etc. I have been using the Dexcom G7 switched over from the G6. It seems the CareSens Air is a bit bigger than the G7. One advantage is it lasts 14 days compared with 10 days of the G7. Warmup time?

**Reply by Ylsani** [reply]
I find it pretty accurate when in range but its often off (showing lower than it really is) when over 220ish. For me thats rare enough occurence that it doesn't really matter, but if you run high often it might be an issue. We don't have g7 or newer libre (still on libre1) in Korea so caresens is smallest. Its similar in size to libre1, tiny bit smaller. Warmup is 2hrs. It also needs daily calibrations (but I find its accurate even without - you can just leave notification there, it works without. If it annoys me I will just put the caresensvalue as calibration. Its usually within 10mg/dl anyway). Bluetooth range is 6m. It has internal memory though so even if it loses connection you will get all values back once it regains it

**Reply by jcisml-G59** [reply]
Got it with thanks. It seems the Dexcom G7 has better stance in every respect, possibly not in its price. Daily calibration requirement sounds sort of absurd.

**Reply by Ylsani** [reply]
Yeah, I think if money doesn't matter G7 might be better option - however, in Korea we don't even have G7 yet anyway (should be coming next year) so I don't know price. G6 is literally double the price of caresens air, so for me it's no brainer, as caresens works for what I need (will G7 be same price as G6 in Korea remains to be seen). I welcome all the competition though, I hope we get like 20 different brands of sensors, the more the better! 

(in regards to calibration though, I think they might update the app to not request it at some point, cause it's not sensor thing, it's app thing, the only truly mandatory calibration is the one when you just apply the sensor)

**Comment by jamorham** [comment]
Official support is likely to be integrated very soon.

**Reply by Ylsani** [reply]
Thank you! I saw they started working on it few weeks ago :) looking forward to it!
